lib/Doctrine/Migrations/Tools/Console/Command/ListCommand.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations\Tools\Console\Command;
+
+use DateTimeInterface;
+use Doctrine\Migrations\Metadata\AvailableMigration;
+use Doctrine\Migrations\Metadata\AvailableMigrationsList;
+use Doctrine\Migrations\Metadata\ExecutedMigration;
+use Doctrine\Migrations\Metadata\ExecutedMigrationsSet;
+use Doctrine\Migrations\Version\Version;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\TableCell;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use function array_map;
+use function array_merge;
+use function array_unique;
+use function uasort;
+
+/**
+ * The StatusCommand class is responsible for outputting a list of all available migrations and their status.
+ */
+class ListCommand extends DoctrineCommand
```

Comment:
Should we make it final?

```suggestion
final class ListCommand extends DoctrineCommand
```